elixir/apps/api/lib/api/client/channel.ex

```diff
@@ -244,38 +244,6 @@ defmodule API.Client.Channel do
     disconnect(socket)
   end
 
-  # FLOWS
```

Comment:
So essentially the pattern is:

- Client channel reacts to _actual_entities being updated/deleted
- If there is an update that changes addressability / routing, we need to toggle the resource down to the client to reset connectivity to a gateway/endpoint
- Gateway channel reacts to flow events